lib/h3core.js

```diff
@@ -843,6 +895,91 @@ function h3Distance(origin, destination) {
     return H3.h3Distance(oLower, oUpper, dLower, dUpper);
 }
 
+/**
+ * Produces IJ coordinates for an H3 index anchored by an origin.
+ *
+ * - The coordinate space used by this function may have deleted
+ * regions or warping due to pentagonal distortion.
+ * - Coordinates are only comparable if they come from the same
+ * origin index.
+ * - Failure may occur if the index is too far away from the origin
+ * or if the index is on the other side of a pentagon.
+ * - This function is experimental, and its output is not guaranteed
+ * to be compatible across different versions of H3.
+ * @static
+ * @param  {H3Index} origin      Origin H3 index
+ * @param  {H3Index} destination H3 index for which to find relative coordinates
+ * @return {Object}              Coordinates as an `{i, j}` pair
+ * @throws {Error}               If the IJ coordinates cannot be calculated
+ */
+function experimentalH3ToLocalIj(origin, destination) {
+    const ij = C._malloc(SZ_COORDIJ);
+    const [oLower, oUpper] = h3IndexToSplitLong(origin);
+    const [dLower, dUpper] = h3IndexToSplitLong(destination);
+    const retVal = H3.experimentalH3ToLocalIj(oLower, oUpper, dLower, dUpper, ij);
```

Comment:
Nit: could also be written as

```js
const retval = H3.experimentalH3ToLocalIj(...h3IndexToSplitLong(origin), ...h3IndexToSplitLong(destination), ij);
```

Since this library assumes a modern JS version and must be babel-ified anyways to run in the browser.

Reply:
True, that's reasonable. Looks like I'm not super consistent about this at the moment.

Reply:
Looking at the code, this is a good suggestion, but I'd rather be consistent about it across the board, so I will postpone this to a later cleanup diff.